src/networking/serialization/mod.rs

```diff
@@ -97,6 +111,28 @@ mod tests {
     // doesnt have anything but needed for the default implementation (as of right now)
   }
 
+// only works with feature(test) and with `extern crate test; use test::Bencher`
+//  #[bench]
```

Comment:
We probably need to decide which testing framework to use libtest or something like Criterion.

Reply:
The bench is only available in `nightly` therefore classified as "unstable" and after quickly looking into it stale as well.

Reply:
We decided to use the extenal benchmarker Criterion.rs